### PR TITLE
Add example for Standalone components and a more functional approach

### PR DESCRIPTION
**Pull request by frederikprijck**
### Description

Add example for Standalone components and a more functional approach 

### References

#442 

### Checklist

- [x] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used, if not the default branch

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -378,3 +379,51 @@ export class AppComponent {
   }
 }
 ```
+
+## Standalone components and a more functional approach
+As of Angular 15, the Angular team is putting standalone components, as well as a more functional approach, in favor of the traditional use of NgModules and class-based approach.
+
+Untill we have built in support for these concepts, you will need to add a couple of small work arounds to leverage these concepts with our SDK:
```
Comment (stevehobbsdev) [inline EXAMPLES.md]:
```suggestion
Until we have built-in support for these concepts, you will need to add a couple of small workarounds to leverage these concepts with our SDK:
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
Looks great, just highlighted a small typo 🎉

**Comment by frederikprijck** [comment]
The implementation here will change once we merge https://github.com/auth0/auth0-angular/pull/445, so moving this to draft again.

**Review by stevehobbsdev — DISMISSED** [review]

File: EXAMPLES.md [hunk 2]
```diff
@@ -378,3 +379,37 @@ export class AppComponent {
...
+
+## Standalone components and a more functional approach
+As of Angular 15, the Angular team is putting standalone components, as well as a more functional approach, in favor of the traditional use of NgModules and class-based approach.
+
+There are a couple of difference with how you would traditionally implement our SDK:
+
+- Use our functional guard (`authGuardFn`) instead of our class-based `AuthGuard`.
+- Use our functional interceptor (`authHttpInterceptorFn`) instead of our class-based `AuthHttpInterceptor`.
+- Register the interceptor by passing it to `withInterceptors` when calling `provideHttpClient`.
+- Register our SDK using `provideAuth`.
```
Comment (frederikprijck) [inline EXAMPLES.md]:
```suggestion
- Register our SDK using `provideAuth0`.
```

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**